common3.json

```diff
@@ -6087,35 +6087,35 @@
     },
     {
       "id" : 61533,
-      "text" : "Flinching: 対象より素早さが高く対象が宿られ状態"
+      "text" : "Flinching: Faster than target and target is seeded"
     },
     {
       "id" : 61534,
-      "text" : "Flinching: 対象より素早さが高く対象が呪い状態"
+      "text" : "Flinching: Faster than target and target is cursed"
     },
     {
       "id" : 61535,
-      "text" : "Flinching: 対象より素早さが高く対象が麻痺状態"
+      "text" : "Flinching: 対Faster than target and target is paralyzed"
```

Comment:
there's a jp character left and it doesn't appear in your list of uncertain one, is this a typo?

Reply:
It was a typo, oops.